2024-07-25 14:25:16,113 ERROR: Erreur lors de la prédiction : [14:25:16] /workspace/src/c_api/../data/array_interface.h:492: Unicode-1 is not supported.
Stack trace:
  [bt] (0) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(+0x15c2ca) [0x79e61af5c2ca]
  [bt] (1) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(+0x19ff29) [0x79e61af9ff29]
  [bt] (2) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(XGDMatrixCreateFromDense+0x16c) [0x79e61af70a6c]
  [bt] (3) /lib/x86_64-linux-gnu/libffi.so.8(+0x7b16) [0x79e6851d5b16]
  [bt] (4) /lib/x86_64-linux-gnu/libffi.so.8(+0x43ef) [0x79e6851d23ef]
  [bt] (5) /lib/x86_64-linux-gnu/libffi.so.8(ffi_call+0x12e) [0x79e6851d50be]
  [bt] (6) /usr/lib/python3.12/lib-dynload/_ctypes.cpython-312-x86_64-linux-gnu.so(+0xe11c) [0x79e6837fd11c]
  [bt] (7) /usr/lib/python3.12/lib-dynload/_ctypes.cpython-312-x86_64-linux-gnu.so(+0x92af) [0x79e6837f82af]
  [bt] (8) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/bin/python(_PyObject_MakeTpCall+0x75) [0x548f55]

 [in /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/api_local.py:51]
2024-07-25 14:25:29,883 INFO: Requête reçue à l'URL : http://0.0.0.0:5000/predict [in /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/api_local.py:39]
2024-07-25 14:25:29,884 ERROR: Erreur lors de la prédiction : [14:25:29] /workspace/src/c_api/../data/array_interface.h:492: Unicode-1 is not supported.
Stack trace:
  [bt] (0) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(+0x15c2ca) [0x79e61af5c2ca]
  [bt] (1) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(+0x19ff29) [0x79e61af9ff29]
  [bt] (2) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(XGDMatrixCreateFromDense+0x16c) [0x79e61af70a6c]
  [bt] (3) /lib/x86_64-linux-gnu/libffi.so.8(+0x7b16) [0x79e6851d5b16]
  [bt] (4) /lib/x86_64-linux-gnu/libffi.so.8(+0x43ef) [0x79e6851d23ef]
  [bt] (5) /lib/x86_64-linux-gnu/libffi.so.8(ffi_call+0x12e) [0x79e6851d50be]
  [bt] (6) /usr/lib/python3.12/lib-dynload/_ctypes.cpython-312-x86_64-linux-gnu.so(+0xe11c) [0x79e6837fd11c]
  [bt] (7) /usr/lib/python3.12/lib-dynload/_ctypes.cpython-312-x86_64-linux-gnu.so(+0x92af) [0x79e6837f82af]
  [bt] (8) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/bin/python(_PyObject_MakeTpCall+0x75) [0x548f55]

 [in /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/api_local.py:51]
2024-07-25 14:29:56,677 INFO: Requête reçue à l'URL : http://0.0.0.0:5000/predict [in /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/api_local.py:39]
2024-07-25 14:29:56,680 ERROR: Erreur lors de la prédiction : [14:29:56] /workspace/src/c_api/../data/array_interface.h:492: Unicode-1 is not supported.
Stack trace:
  [bt] (0) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(+0x15c2ca) [0x6fff0b55c2ca]
  [bt] (1) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(+0x19ff29) [0x6fff0b59ff29]
  [bt] (2) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/lib/python3.12/site-packages/xgboost/lib/libxgboost.so(XGDMatrixCreateFromDense+0x16c) [0x6fff0b570a6c]
  [bt] (3) /lib/x86_64-linux-gnu/libffi.so.8(+0x7b16) [0x6fff767c8b16]
  [bt] (4) /lib/x86_64-linux-gnu/libffi.so.8(+0x43ef) [0x6fff767c53ef]
  [bt] (5) /lib/x86_64-linux-gnu/libffi.so.8(ffi_call+0x12e) [0x6fff767c80be]
  [bt] (6) /usr/lib/python3.12/lib-dynload/_ctypes.cpython-312-x86_64-linux-gnu.so(+0xe11c) [0x6fff75b2811c]
  [bt] (7) /usr/lib/python3.12/lib-dynload/_ctypes.cpython-312-x86_64-linux-gnu.so(+0x92af) [0x6fff75b232af]
  [bt] (8) /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/.venv/bin/python(_PyObject_MakeTpCall+0x75) [0x548f55]

 [in /home/tamara-daniel-tricot/Bureau/0_Projets_ParcoursDataScientist/P7/Projet_7/api_local.py:51]
